code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -399,7 +401,37 @@ func (edb *EventDb) WorkAggregates(
 func (edb *EventDb) managePartitions(round int64) {
 	logging.Logger.Info("managing partitions", zap.Int64("round", round))
 	edb.AddPartitions(round)
-	edb.dropPartitions(round)
+	edb.movePartitions(round)
+}
+
+func (edb *EventDb) movePartitions(round int64) {
+	if err := edb.movePartitionToSlowTableSpace(SLOWTABLESPACE, "events"); err != nil {
```

Comment:
it's fine to delete this ones, we need to move only `transactions` and `blocks` to an old tablespace